rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/ConfigSummary.java

```diff
@@ -114,7 +116,8 @@ public ConfigSummary(ConfigKey<?> config, String label, Double priority, Boolean
                 : ImmutableList.<String>of();
         if (config.getType().isEnum()) {
             this.type = Enum.class.getName();
-            this.defaultValue = (config.getDefaultValue() == null) ? null : ((Enum<?>) config.getDefaultValue()).name();
+            this.defaultValue = config.getDefaultValue() instanceof Enum? ((Enum<?>) config.getDefaultValue()).name() : 
+                Jsonya.convertToJsonPrimitive(config.getDefaultValue());
```

Comment:
Same a above. Also, not sure I understand why converting to JSON primitive here?

Reply:
For consistency with code below, when it isn't an enum -- we always do that in case the default value is something complex (which it shouldn't be unless maybe a DSL but consistency with code below).